### PR TITLE
 Move metrics to remotedialer

### PR DESCRIPTION
**Pull request by dramich**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,6 +6,6 @@ require (
 	github.com/gorilla/mux v1.7.3
 	github.com/gorilla/websocket v1.4.0
 	github.com/pkg/errors v0.8.1
-	github.com/rancher/norman v0.0.0-20191118171605-5a9f64e84e65
+	github.com/prometheus/client_golang v1.4.0
```
Comment (ibuildthecloud) [inline go.mod]: Can you put this to 	github.com/prometheus/client_golang v1.0.0 to match what is in norman?

**Comment by ibuildthecloud** [comment]
LGTM

File: go.mod (hunk 1 above)
Comment (dramich) [reply go.mod]: Norman is getting bumped to v1.4.0

File: metrics/session_manager.go [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+			Subsystem: "session_server",
+			Name:      "total_peer_ws_disconnected",
+			Help:      "Total count of disconnected websocket sessions from other rancher-server",
+		},
+		[]string{"peer"},
+	)
+)
+
+func init() {
+	if os.Getenv(metricsEnv) == "true" {
```
Comment (alexellis) [inline metrics/session_manager.go:102]: How are these metrics served at runtime?

**Comment by dramich** [comment]
@alexellis reaching out on Rancher user slack

**Pull request merged**